src/CMakeLists.txt

```diff
@@ -403,6 +403,73 @@ if(WITH_DPDK)
   list(APPEND ceph_common_deps common_async_dpdk)
 endif()
 
+if(WITH_PMDK)
+  # used in src/librbd/CMakeLists.txt and src/os/CMakeLists.txt
+  if(WITH_PMDK_PKG)
+    # Use installed libpmem-dev and libpmemobj-dev. This will make
+    # libpmem1 and libpmemobj1 build dependencies for Ceph, and install dependencies for
+    # anything that links to them (librbd).
+    find_package(libpmem-dev)
```

Comment:
i don't think this would work. as neither cmake ships `Findlibpmem-dev.cmake`, nor we have this file. i'd suggest extract this whole chunk into `Buildpmem.cmake` and `Findpmem.cmake`.

Reply:
I tested this point, it works.

Reply:
i think the reason it "works" is just `REQUIRED` is not passed to this function. please see https://cmake.org/cmake/help/v3.0/command/find_package.html for more details.

again, please extract this whole chunk into `Buildpmem.cmake` and `Findpmem.cmake`.

Reply:
Done